### DISCUSSION TITLE
is there a way to export an audio stream to .wav

### DISCUSSION THREAD

**Discussion by cyberic99** [raysan5/raylib#2502] [Q&A]
Hi,
I have an application where I use raylib to play several sounds simultaneously, with various volumes and pitches.

I can do in in realtime, but I'd like to know if there is a way to make the time advance myself, manually call the audio callback, and export the mixed result as a .wav for instance

Thanks for your answers

**Comment by jdekoto** [comment]
i was looking for the same thing buddy. we might need to extract the exporter from rFXGen
